src/converter.ts

```diff
@@ -126,6 +127,10 @@ async function createConverter(converterType: string): Promise<AbstractConverter
             console.log("[i] Processing file using XTB converter");
             converter = new XtbConverter(securityService);
             break;
+        case "investimental":
```

Comment:
Could you move this up, so it is alphabetically sorted?

Reply:
done